bin/e2e/update-e2e-check-run-status.ts

```diff
@@ -0,0 +1,57 @@
+import fetch from "node-fetch";
+
+async function updateE2eCheckRunStatus(): Promise<void> {
+  if (process.env.GITHUB_CONTEXT_EVENT === undefined) {
+    throw new Error("GITHUB_CONTEXT_EVENT environment variable is undefined");
+  }
+
+  const githubActionsContext = JSON.parse(process.env.GITHUB_CONTEXT_EVENT);
+
+  if (process.env.GITHUB_TOKEN === undefined) {
+    throw new Error("GITHUB_TOKEN environment variable is undefined");
+  }
+
+  const hasDeploymentFailed =
+    githubActionsContext.deployment_status.state === "failure";
+
+  const checkSuiteList = await fetch(
```

Comment:
If the deployment has failed, do we need to get the checks or can we immediately update it to failure?

Reply:
I think we need to get the checks anyway, because we need to retrieve the custom check id to update its status (to construct the URL on which we're going to do the PATCH thing). 

In fact, I spend some time on thinking how to "pass" this id between workflows (`lint-and-test.yml` is where we have it in the create check run script) but I concluded it will be more simple to fetch all checks (since we need to do it to retrieve the one concerning real e2e tests if deployment is ok anyway)

Reply:
Forgot about that, you're right 👍